src/routing/comm.rs

```diff
@@ -406,12 +413,13 @@ mod tests {
     #[tokio::test]
     async fn send_after_reconnect() -> Result<()> {
         let (tx, _rx) = mpsc::channel(1);
-        let send_comm = Comm::new(transport_config(), tx)?;
+        let send_comm = Comm::new(transport_config(), tx).await?;
 
         let recv_transport = QuicP2p::with_config(Some(transport_config()), &[], false)?;
-        let recv_endpoint = recv_transport.new_endpoint()?;
-        let recv_addr = recv_endpoint.socket_addr().await?;
-        let mut recv_incoming_connections = recv_endpoint.listen();
+        #[allow(unused)]
```

Comment:
Better prefix the unused variables with underscore, so it's obvious which are actually unused.

Reply:
This one sneaked in, my bad :)